test/helpers/json-reporter.ts

```diff
@@ -0,0 +1,68 @@
+import { existsSync, readFileSync, writeFileSync } from 'fs';
+import reporter, { SuiteStats, TestStats } from '@wdio/reporter';
```

Comment:
I think `reporter` should be capitalised

Reply:
Updated!